ACID/acid_wotlk.sql

```diff
@@ -12560,9 +12560,12 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 -- Falgran Hastil
 ('508801','5088','2','0','100','0','15','0','0','0','0','0','25','0','0','0','1','-47','0','0','0','0','0','0','Falgran Hastil - Flee at 15% HP'),
 -- Balos Jacken
-('508901','5089','2','0','100','0','20','0','0','0','0','0','1','-775','0','0','24','0','0','0','22','1','0','0','Balos Jacken - Say and Evade and Set Phase 1 at 20% HP'),
-('508902','5089','21','5','100','0','0','0','0','0','0','0','2','35','0','0','22','2','0','0','0','0','0','0','Balos Jacken - Become Friendly and Set Phase 2 on Reached Home (Phase 1)'),
-('508903','5089','1','3','100','1','120000','120000','120000','120000','0','0','2','54','0','0','22','0','0','0','0','0','0','0','Balos Jacken - Become Hostile After 2 Minutes OOC and Set Phase 0 (Phase 2)'),
+('508901','5089','11','0','100','0','0','0','0','0','0','0','42','1','0','0','17','82','0','0','0','0','0','0','Balos Jacken - Set Invicible and Set NpcFlag 0 on Spawn'),
+('508902','5089','2','0','100','0','20','0','0','0','0','0','2','84','0','0','18','768','0','0','22','1','0','0','Balos Jacken - Set Faction Alliance Generic and Set UnitFlags 256+512 and Set Phase 1 at 20% HP'),
+('508903','5089','2','0','100','0','20','0','0','0','0','0','24','0','0','0','0','0','0','0','0','0','0','0','Balos Jacken - Evade at 20% HP'),
+('508904','5089','21','5','100','0','0','0','0','0','0','0','1','-775','0','0','17','82','2','0','22','2','0','0','Balos Jacken - Say and Set NpcFlag 2 and Set Phase 2 on Reached Home (Phase 1)'),
```

Comment:
Small note here and I will correct it on the wiki right away. Its a bit annoying but these are marked deprecated because they were a major source of issues across the 3 versions. Use a relay script and set the flags using either SCRIPT_COMMAND_MODIFY_NPC_FLAGS or SCRIPT_COMMAND_MODIFY_UNIT_FLAGS. Also do not use toggle on these commands.

There are multiple reasons to unpack why. Firstly, these commands are safe to port as-is across our 3 versions unlike the offseted ones. Two, we are trying to somewhat unify EAI and dbscript scripting so that we do not have duplicated actions in each (easier to maintain even if a little more annoying to script). Thirdly, toggle is problematic because if the script breaks due to some outside factors, toggle will not self

Reply:
Small note, I got a bit trigger happy, only EAI action 17 is deprecated, 18 and 19 are fine.

Reply:
That makes sense, I'll change it. Thanks for the heads-up.

Reply:
Done.